### ISSUE TITLE
Ability to change CPU registers from the debugger

### ISSUE THREAD

**Issue by NovaSquirrel** [NovaSquirrel/Mesen-S#8]
It's an editable text box but it doesn't actually change the register if you modify it.

**Comment by lab313ru** [comment]
Will commit today a code that changes registers from the core and interop. Just apply it:)

**Comment by lab313ru** [comment]
https://github.com/NovaSquirrel/Mesen-S/pull/12

**Comment by lab313ru** [comment]
Branch https://github.com/NovaSquirrel/Mesen-S/tree/ui_regs_change now have an ability to change CPU registers (commit 27c6df32c3e18577c73a2ee1644cb86a24c2557d).

**Comment by lab313ru** [comment]
To change registers of PPU, etc., it requires changing from ListView to something else.

**Comment by NovaSquirrel** [comment]
You mean to add it to the Register Viewer? We could do that, probably not as high priority as having it for CPU registers though.

**Comment by lab313ru** [comment]
Yes, I mean Register Viewer. It's readonly now. ListView doesn't allow editing.
